### DISCUSSION TITLE
Wireguard

### DISCUSSION THREAD

**Discussion by Jenstel** [hufrea/byedpi#121] [General]
Можно ли ваше решение использовать для обхода блокировки Wireguard, чтобы отправлять фейковые udp?

**Comment by hufrea** [comment]
Да, на Android это можно сделать в NekoBox. На десктопе, конечно, можно заморочиться с tun2socks/redsocks + wireproxy, но лучше использовать Zapret.

**Reply by zimbabwe303** [reply]
Пробовал похожим способом увеличить скорость на OpenVPN пущенном через SSL туннель, использовал proxychains, но это ничего дало вообще - скорость режется очень сильно по-прежнему. Поэтому забил на эти эксперименты и поставил AmneziaWG ([amnezia-tools](https://github.com/amnezia-vpn/amneziawg-tools) + [amnezia-go](https://github.com/amnezia-vpn/amneziawg-go)), в коротом вставка фейков добавлена прямо в сам клиент.
